docs/home/resources.md

```diff
@@ -0,0 +1,31 @@
+# PWA Development Resources
+
+If you are looking for more information about building progressive web apps, here are some good places to start:
+
+## *30 Days of PWA*
+
+*30 Days of PWA* is a blog series that teaches progressive web app concepts from the very basics to more advanced practices. It's a great place to start if you're looking for some more in depth explanation of PWA concepts.
```

Comment:
> "It's a great place to start if you're looking for some more in depth explanation of PWA concepts."

Doesn't read well to me, I think instead of "some" just say "a"

Reply:
```suggestion
*30 Days of PWA* is a blog series that teaches progressive web app concepts from the very basics to more advanced practices. It's a great place to start if you're looking for a more in depth explanation of PWA concepts.
```